File: docs/apis/reference/snippets/java/a2v/video_generate.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
MediaType mediaType = MediaType.parse("application/json");
RequestBody body = RequestBody.create(mediaType, "{\n    \"model_id\": \"eva_a0008\",\n    \"audio_url\": \"https://example.com/audio.mp3\"\n}");
Request request = new Request.Builder()
  .url("https://openapi.surreal-ai.com/v1/a2v/video.generate")
  .method("POST", body)
  .addHeader("Authorization", "bee03bed6352400383d9a51ba325af2f")
  .addHeader("Content-Type", "application/json")
  .build();
Response response = client.newCall(request).execute();

File: docs/apis/reference/snippets/golang/a2v/video_generate.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"strings"
)

func main() {

	url := "https://openapi.surreal-ai.com/v1/a2v/video.generate"
	method := "POST"

	payload := strings.NewReader(`{
    "model_id": "eva_a0008",
    "audio_url": "https://example.com/audio.mp3"
}`)

	client := &http.Client{}
	req, err := http.NewRequest(method, url, payload)

	if err != nil {
		fmt.Println(err)
		return
	}
	req.Header.Add("Authorization", "bee03bed6352400383d9a51ba325af2f")
	req.Header.Add("Content-Type", "application/json")

	res, err := client.Do(req)
	if err != nil {
		fmt.Println(err)
		return
	}
	defer res.Body.Close()

	body, err := ioutil.ReadAll(res.Body)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(string(body))
}
